server/routes/employer.routes.js

```diff
@@ -14,15 +14,15 @@ router.post('/',
         .not()
         .isEmpty()
         .withMessage('Please enter your first name')
-        .isAlpha()
+        .isAlphanumeric('en-US',{ignore: ' '})
         .withMessage('Only characters are allowed for your first and last name')
         .isLength({ min: 2, max: 30})
         .withMessage('A min length of 2 characters and a max of 30 characters is required for your first and last name'),
     check('lastName', 'Appropriately enter your last name with a min of 2 characters and a max of 30')
         .not()
         .isEmpty()
         .withMessage('Please enter your last name')
-        .isAlpha()
+        .isAlphanumeric('en-US',{ignore: ' '})
```

Comment:
I was thinking of putting this for organization but I guess putting them in first / last name is fine, but you left out the org haha

Reply:
I was thinking that some orgs would have numbers, like Studio 86 or something. So I left it out for now and just looked at blanks. Something we can discuss next week maybe?